bin/deploy/try-tome-enable

```diff
@@ -8,8 +8,22 @@ if [ x$1 == x"--dryrun" ]; then
   shift
 fi
 
+SCRIPT_DIR="$( cd -- "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )"
+if [ -f $SCRIPT_DIR/includes ]; then
+  . $SCRIPT_DIR/includes
+fi
+
+needsDisabled=0
```

Comment:
I will take a look - it should be executing the same logic that the try-tome-disable script uses (e.g. if ssh needs enabled, then it will also need disabled).